open_instruct/grpo_fast.py

```diff
@@ -2559,21 +2579,21 @@ def maybe_evaluate(
             logger.info("[Main Thread] didn't get eval generation metrics")
 
         eval_sequence_lengths = np.array([len(response) for response in eval_result.responses])
-        eval_decoded_responses = tokenizer.batch_decode(eval_result.responses, skip_special_tokens=True)
-        eval_stop_rate = sum(int(finish_reason == "stop") for finish_reason in eval_result.finish_reasons) / len(
-            eval_result.finish_reasons
-        )
-
-        # get and log evaluation metrics
-        eval_scores, eval_reward_metrics = asyncio.run(
-            reward_fn(
-                eval_result.responses,
-                eval_decoded_responses,
-                eval_batch if eval_batch else Batch(queries=[], ground_truths=[], datasets=[], indices=None),
-                eval_result.finish_reasons,
-                eval_result.request_info,
-            )
-        )
+        # eval_decoded_responses = tokenizer.batch_decode(eval_result.responses, skip_special_tokens=True)
```

Comment:
Why comment these out?

Reply:
this should be deleted not commented out, it was deleted in a later commit, we're doing all of this stuff in accumulate_inference_batches